mantle/kola/harness.go

```diff
@@ -924,10 +977,104 @@ func RegisterExternalTests(dir string) error {
 	return RegisterExternalTestsWithPrefix(dir, basename)
 }
 
+// Create a parent test that runs non-exclusive tests as subtests
+func makeNonExclusiveTest(tests []*register.Test, config *conf.UserData, flight platform.Flight) (register.Test, func(h *harness.H)) {
```

Comment:
I don't follow this; I would expect it to return just a `register.Test` whose `Run` runs all the non-exclusive tests as subtests. Then we could use the same loop where we have the `runTest` closure.